Source/ThirdParty/ANGLE/src/tests/gl_tests/UniformBufferTest.cpp

```diff
@@ -2801,6 +2801,78 @@ TEST_P(UniformBlockWithOneLargeArrayMemberTest, MemberTypeIsMatrix)
     checkResults(GLColor::green, GLColor::red, GLColor::green, GLColor::green);
 }
 
+#if 0
```

Comment:
This looks like an `#ifdef`'d out copy of the previous test. Would you remove it?

Reply:
This isn't supposed to be in this change. Reverting.